configure.ac

```diff
@@ -74,14 +74,11 @@ AS_IF([test "x$enable_gtk3" = xyes],
 AM_CONDITIONAL([GTK3], [test "x$gtk_package" = "xgtk+-3.0"])
 
 # GTK/GLib/GIO checks
-gtk_modules="$gtk_package >= $gtk_min_version glib-2.0 >= 2.28"
-gtk_modules_private="gio-2.0 >= 2.28 gmodule-no-export-2.0"
+gtk_modules="$gtk_package >= $gtk_min_version glib-2.0 >= 2.32"
+gtk_modules_private="gio-2.0 >= 2.32 gmodule-no-export-2.0"
```

Comment:
Ah, it's the MinGW GTK2 build [that downloads the 2.24 bundle](https://travis-ci.org/geany/geany/jobs/135397307#L551) which presumably doesn't have a new enough GLib version in it.

This might genuinely be a problem on Windows if we're still supporting pre-built GTK2 bundle (ex. makefile.win32 files or something). It would be easy to fix it though by just replacing `G_TYPE_KEYFILE` with `G_TYPE_POINTER` or whatever, since it's an opaque type, it shouldn't matter ABI-wise or anything. If `g-ir-scanner` has troubles, maybe it could be `#ifdef`'d back to `G_TYPE_KEYFILE` for when being scanned by that tool.